com.github.imagefanreloaded.json

```diff
@@ -0,0 +1,37 @@
+{
+    "app-id" : "io.github.mihnearadulescu.imagefanreloaded",
```

Comment:
the github username for the repository doesn't match.

Reply:
Done, thanks.

Reply:
this what not what I was saying